platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/PropertyFactory.java

```diff
@@ -3364,6 +3364,7 @@
     return new LayoutPropertyValue<>("text-optional", value);
   }
 
+
```

Comment:
What introduced this extra newline?

Reply:
Unsure, but that's what the code generator produces. Might be some change in the template, or someone forgot to commit this newline from the generator, e.g. in https://github.com/mapbox/mapbox-gl-native/commit/757cc0f2be1c0972f9b74a91ed873fe8a892f27f